### ISSUE TITLE
Generator PR

### ISSUE THREAD

**Issue by fotiskoun** [fotiskoun/CCF#38]
Add the generator files in a PR to CCF

**Comment by heidihoward** [comment]
Requires to moving current commented out code (for LSKV) into a separate file, see discussion in https://github.com/fotiskoun/CCF/issues/15

**Comment by fotiskoun** [comment]
Yes this issue is blocked by https://github.com/fotiskoun/CCF/issues/44

**Comment by heidihoward** [comment]
I believe this is now unblocked. Do you want to tidy the generator code up and open a PR to CCF?

**Comment by fotiskoun** [comment]
Yes, also I was thinking to PR the ansible and CMake things that were separate. But I didn't search or ask again about the build container I thought Amaury mentioned for the PR. I have a look again tomorrow and might ask Eddy or Amaury

**Comment by heidihoward** [comment]
I think ansible/CMake should be a separate issue and seperate PR for CCF. The generator is just python so should not depend on these

**Comment by heidihoward** [comment]
This PR has been created: https://github.com/microsoft/CCF/pull/4429

**Comment by heidihoward** [comment]
https://github.com/microsoft/CCF/pull/4429 has been merged into main.
